optimum/habana/transformers/models/clip/modeling_clip.py

```diff
@@ -60,7 +60,7 @@ def __init__(self):
         super().__init__()
 
     def forward(self, x, dim=None, invAttnHead=None):
-        return torch.ops.hpu.softmax_fp8(x, dim, None, None, invAttnHead)
```

Comment:
@tthakkal there are other instances of softmax_fp8 in https://github.com/huggingface/optimum-habana/blob/0027e320b3f6c3f8a98d733ae9a42901de8a1e9d/optimum/habana/diffusers/models/attention_processor.py#L42 & https://github.com/huggingface/optimum-habana/blob/0027e320b3f6c3f8a98d733ae9a42901de8a1e9d/optimum/habana/transformers/models/falcon/modeling_falcon.py#L139

probably we can update those here

Reply:
first one is nowhere used, second one runs only for fp8 use case, so not impacted.